alongtheway.txt

```diff
@@ -632,7 +632,7 @@ In Luke Jesus rebuked the disciples for their lack of faith that He had risen fr
 Day 5: becoming
 Read: John 14:15-21, 16:5-16, Acts 1:8, 2:14-41
 
-Reflect: When Peter walked with Jesus he was dependent on Him. The moment Jesus was taken away Peter became unsure and defeated. (Matt. 26:69-75) That is why we need the Holy Spirit, the power of Jesus, living inside of us. When He is with us all of the time we are more capable to do good (the will of God) than we could ever imagine. The Holy Spirit in us draws people to Jesus.
+Reflect: When Peter walked with Jesus he was dependent on Him. The moment Jesus was taken away Peter became unsure and defeated. (Matt. 26:69-75) That is why we need the Holy Spirit, the power of Jesus, living inside of us. When He remains with us all of the time, we are more capable to do good (the will of God) than we could ever imagine. The Holy Spirit in us draws people to Jesus.
```

Comment:
"Dwelling inside of us...." instead of "remains...."
On Jan 15, 2016 11:18 PM, "Joshua Maxwell" <notifications@github.com> wrote:

> In alongtheway.txt
> <https://github.com/joshuamaxwell/alongtheway/pull/11#discussion_r49927086>
> :
>
> > @@ -632,7 +632,7 @@ In Luke Jesus rebuked the disciples for their lack of faith that He had risen fr
> >  Day 5: becoming
> >  Read: John 14:15-21, 16:5-16, Acts 1:8, 2:14-41
> >
> > -Reflect: When Peter walked with Jesus he was dependent on Him. The moment Jesus was taken away Peter became unsure and defeated. (Matt. 26:69-75) That is why we need the Holy Spirit, the power of Jesus, living inside of us. When He is with us all of the time we are more capable to do good (the will of God) than we could ever imagine. The Holy Spirit in us draws people to Jesus.
> > +Reflect: When Peter walked with Jesus he was dependent on Him. The moment Jesus was taken away Peter became unsure and defeated. (Matt. 26:69-75) That is why we need the Holy Spirit, the power of Jesus, living inside of us. When He remains with us all of the time, we are more capable to do good (the will of God) than we could ever imagine. The Holy Spirit in us draws people to Jesus.
>
> switch "when he is with us all the time" with "when He remains with us"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joshuamaxwell/alongtheway/pull/11/files#r49927086>.
>